src/goto-analyzer/static_verifier.h

```diff
@@ -40,7 +42,20 @@ void static_verifier(
 enum class ai_verifier_statust { TRUE, FALSE, BOTTOM, UNKNOWN };
 // clang-format on
 
-std::string as_string(ai_verifier_statust);
-
+std::string as_string(const ai_verifier_statust &);
```

Comment:
This change perhaps belongs in the prior commit?

Reply:
Ooops!  Fixed.